cmd/kubeadm/app/phases/upgrade/compute.go

```diff
@@ -299,11 +317,13 @@ func minorUpgradePossibleWithPatchRelease(stableVersion, patchVersion *versionut
 	return patchVersion.LessThan(stableVersion)
 }
 
-func getSuggestedEtcdVersion(kubernetesVersion string) string {
-	etcdVersion, err := kubeadmconstants.EtcdSupportedVersion(kubernetesVersion)
+func getSuggestedEtcdVersion(kubernetesVersion string) (string, error) {
+	etcdVersion, warning, err := kubeadmconstants.EtcdSupportedVersion(kubeadmconstants.SupportedEtcdVersion, kubernetesVersion)
 	if err != nil {
-		fmt.Printf("[upgrade/versions] WARNING: No recommended etcd for requested Kubernetes version (%s)\n", kubernetesVersion)
-		return "N/A"
+		return "N/A", errors.Wrap(err, "Couldn't retrieve an etcd version for the target Kubernetes version")
+	}
+	if warning != nil {
+		fmt.Printf("[upgrade/versions] %v\n", warning)
```

Comment:
i think we should use `klog.Warningln(...)` consistently for printing the warning.